### PR TITLE
Implement tests for Room's fields

### PR DESCRIPTION
**Pull request by DrWala**
### What this does:
- Implement tests for the fields of Room

### How to test:
- Checkout the PR and run the tests. All tests should pass.
- Observe code coverage increase in jacoco test report for model room

### CODE REVIEW

File: src/test/java/seedu/address/model/room/RoomNumberOrTagsContainsKeywordsPredicateTest.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        assertFalse(predicate.test(new RoomBuilder().withTags("tagOne").build()));
+
+        // Non-matching keyword
+        predicate = new RoomNumberOrTagsContainsKeywordsPredicate(Arrays.asList("tagOne", "tagTwo"));
+        assertFalse(predicate.test(new RoomBuilder().withTags("tagThree").build()));
+
+        // Keywords match occupancy and room type, but does not match room number
+        predicate = new RoomNumberOrTagsContainsKeywordsPredicate(Arrays.asList("suite_ac", "n"));
+        assertFalse(predicate.test(new RoomBuilder()
+                .withRoomNumber("03-345")
```
Comment (stein414) [inline src/test/java/seedu/address/model/room/RoomNumberOrTagsContainsKeywordsPredicateTest.java:117]: Probably better to use the static variables from the typical rooms class

**Review by stein414 — APPROVED** [review]
Just a comment but LGTM.

File: src/test/java/seedu/address/model/room/RoomNumberOrTagsContainsKeywordsPredicateTest.java (hunk 1 above)
Comment (DrWala) [reply src/test/java/seedu/address/model/room/RoomNumberOrTagsContainsKeywordsPredicateTest.java:117]: Will update.

Comment (DrWala) [reply src/test/java/seedu/address/model/room/RoomNumberOrTagsContainsKeywordsPredicateTest.java]: Updated.

**Pull request merged**